sklearn_numba_dpex/kmeans/drivers.py

```diff
@@ -156,7 +156,11 @@ def kmeans(
         number_of_load_iter=4,
     )
 
-    X = dpctl.tensor.from_numpy(np.ascontiguousarray(X).T, device=device)
+    # The dpex kernel expects X to be shaped as (n_features, n_samples) as the
+    # usual (n_samples, n_features). Furthermore, using a Fortran memory layout
```

Comment:
```suggestion
    # The dpex kernel expects X to be shaped as (n_features, n_samples) instead of
    # the usual (n_samples, n_features). Furthermore, using a Fortran memory layout
```